### ISSUE TITLE
Can't access indivual project notifcations

### ISSUE THREAD

**Issue by AurySystem** [valknight/Cohost.py#21]
The api endpoint "notifications/list" seems to require more data to fetch a project's notification feed and only fetches the list from the defualt project

**Comment by valknight** [comment]
🤦 yeah that makes sense. this shouldn't be too hard a one to fix up, thank you!

**Comment by valknight** [comment]
Ok, project switching should be included in the next release. I'll leave this open, but, as a lower priority now this is unblocked.
